style/index.css

```diff
@@ -0,0 +1 @@
+@import url('base.css');
```

Comment:
What is the reason for importing an empty file?

Reply:
Thanks for catching this was a consequence of applying the upgrade extension script. But in this case this is indeed not needed.